.vib/appsmith/cypress/cypress/integration/appsmith_spec.js

```diff
@@ -5,30 +5,23 @@ import {
 
 it('allows to create a new project', () => {
   cy.login();
-  cy.get('[class*="createnew"]').click();
-  // We don't need a name as it already adds a random name
-  cy.contains('Widgets');
-  // There is a pop-up (when there is a new version available) that takes some
-  // animation to appear, but when it does, it blocks a button. We need to wait
-  // a few seconds to let it appear and then remove it so we can continue
+  // Go to the templates page
+  cy.get('[class*="templates-tab"]').click();
+
   cy.get('body').then(($body) => {
     // Close the pop-up if appears
     if ($body.find('[class*="toast-action"]').is(':visible')) {
       cy.get('[class*="toast-action"]').click();
     }
   });
-  cy.contains('template').click({force: true});
+  // Create an application from the Marketing Portal template
   cy.contains('[data-cy="template-card"]', 'Marketing Portal').within(() => {
     cy.get('[class*="fork-button"]').click();
   })
-  cy.contains('Marketing Portal');
-  cy.contains('Deploy').click();
-  // This will open a new window, but we can use the following workaround to see the page
-  cy.get('[class*="deploy-popup"]').click();
-  cy.get('[class*="current-deployed"]').invoke('removeAttr', 'target').click()
-  // We check that the expected exist but also a button not present in the edition UI
-  cy.contains('Marketing Portal');
-  cy.contains('Edit App');
+  cy.contains('FORK TEMPLATE').click();
```

Comment:
Is there any kind of message after an application has been created? It would be good to look for it before going to `/applications`; I fear we are introducing a race condition here. 

Additionally, could you include a link to a successful run of these tests? You can also temporarily bump appsmith chart version in this PR so VIB is launched. For this, take care of #14862, which was landed the other day, and modified the same files.

Reply:
I've added a check for "Deploy", which appears after the project has been created. I've also bumped the version to trigger the tests.